update create-azure-plan.md (#53)
MicrosoftDocs/partner-center-sdk: develop/create-azure-plan.md

For details on how to disable Azure usage subscriptions, see [Azure API on subscription lifecycle management](https://docs.microsoft.com/rest/api/resources/subscriptions).

> To remove existing Azure reservations, you must submit a [cancellation request](https://docs.microsoft.com/partner-center/azure-reservations-manage#cancel-or-exchange-a-reservation). Once an Azure plan is suspended, you can reactivate it.

Reviewer: ```suggestion
To remove existing Azure reservations, you must [cancel the reservations](azure-reservations-manage.md#cancel-or-exchange-a-reservation).  After you suspend an Azure reservation, you can reactivate it.
```

> For details on how to reactivate an Azure plan, see [reactivate-a-suspended-a-subscription](https://docs.microsoft.com/en-us/partner-center/develop/reactivate-a-suspended-a-subscription)

Reviewer: ```suggestion
For details on how to reactivate an Azure plan, see [Reactivate a suspended subscription](reactivate-a-suspended-a-subscription.md)
```
